### PR TITLE
Avoid errors when installing with opam

### PR DESCRIPTION
**Pull request by imbsky**

### CODE REVIEW

**Comment by cristianoc** [comment]
@imbsky thanks opam pin works now.

Not planning on pasting the esy path everywhere though. It should be possible to create the symbolic link instead of checking it in, and opam should be undisturbed.

Also, I prefer to keep `reanalyze.exe` as that's then a single name that is used on Windows too. There are also publishing scripts that might depend on that.

**Comment by imbsky** [comment]
I reset it to the minimum amount. I will continue this when I have time. or you should be able to operate my branch, so please feel free to make changes.

File: dune [hunk 1]
```diff
@@ -1 +1 @@
-(dirs :standard \ node_modules)
+(dirs :standard \ examples node_modules)
```
Comment (cristianoc) [inline dune:1]: That's a pretty simple change if it works!

**Comment by imbsky** [comment]
Close this once.

**Comment by cristianoc** [comment]
@imbsky done here: https://github.com/cristianoc/reanalyze/pull/56

**Comment by imbsky** [comment]
Awesome! Thank you.

**Pull request closed without merging**